docs/developers-clear-containers-install.md

```diff
@@ -18,122 +34,70 @@ process.
   * [Proxy](https://github.com/clearcontainers/proxy)
   * [Shim](https://github.com/clearcontainers/shim)
 
+Since the installation guide will have installed packaged versions of
+all required components, it is only necessary to install the source for the
+component(s) you wish to develop with.
+
 **IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers).
 Both projects ship ``cc-proxy`` and they are not compatible with each other.
 
-## Setup the environment
+### Setup the environment
 
 1. Define GOPATH
 
-```bash
-$ export GOPATH=$HOME/go
-```
+   ```bash
+   $ export GOPATH=$HOME/go
+   ```
 
 2. Create GOPATH Directory
 
-```bash
-$ mkdir -p $GOPATH
-```
+   ```bash
+   $ mkdir -p $GOPATH
+   ```
 
 3. Get the code
 
-```bash
-$ go get -d github.com/clearcontainers/runtime
-$ go get -d github.com/clearcontainers/proxy
-$ git clone https://github.com/clearcontainers/shim $GOPATH/src/github.com/clearcontainers/shim
-$ go get -d github.com/clearcontainers/tests
-```
+   ```bash
+   $ go get -d github.com/clearcontainers/runtime
+   $ go get -d github.com/clearcontainers/proxy
+   $ git clone https://github.com/clearcontainers/shim $GOPATH/src/github.com/clearcontainers/shim
+   $ go get -d github.com/clearcontainers/tests
```

Comment:
missing agent?

Reply:
I thought about that. The problem with adding the agent on this PR (which is just to update what we already have) is that we'll then need to explain how to install it into an image (it won't be just `sudo make install` like the rest) and I don't think even the osbuilder docs currently cover that.

I think we probably need a doc in the agent repo to explain so that _this_ document can reference it.

@sameo, @grahamwhaley, @jcvenegas - what do you think?

Reply:
@sameo, @devimc, @grahamwhaley, @jcvenegas - I've added more details for the agent, including osbuilder details although it's debatable whether some of this should live in osbuilder's README instead. Thoughts?

Reply:
@jodh-intel nice!,  IMO it should be part of osbuilder  or an special section to build CC artifacts.

Reply:
I suspect the info on how to install the agent is probably best placed in the osbuilder repo, and then we can reference that from both here and in the agent repo itself (the agent repo is somewhat lacking in any docs/details :-)
Having said that, for now we can put the info here and move it later to the osbuilder repo - no problem doing that if we need expediency.